extensions/mssql/src/tableDesigner/tableDesigner.ts

```diff
@@ -0,0 +1,31 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { AppContext } from '../appContext';
+import * as azdata from 'azdata';
+import * as vscode from 'vscode';
+import { sqlProviderName } from '../constants';
+
+export function registerTableDesignerCommands(appContext: AppContext) {
```

Comment:
if TableDesigner is "built-in" feature commands should probably be in the core.  If it's generic extension I guess we'd create a new extension for this, though not sure that's the right design.  Not sure why we'd want the commands here.